spec/system/decidim/blogs/author_manages_blogpost_spec.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe "Admin manages posts", type: :system do
```

Comment:
```suggestion
describe "Author manages blog posts", type: :system do
```
Posts management is done by Admin or Author ? 
If user doesn't have to be Admin to manage I think we should specify in `describe`  clause, otherwise forget about this comment